source/Nuke.Common/Tooling/ToolPathResolver.cs

```diff
@@ -67,7 +69,7 @@ string GetFramework(string file)
 
         private static string GetPackageDirectory(string[] packageIds, string version)
         {
-            return packageIds
```

Comment:
I would wrap the original statement with a try-catch... Then rethrow with the message modified.

Reply:
done as suggested.